.circleci/.anchore/grype.yaml

```diff
@@ -0,0 +1,16 @@
+ignore:
+
+  # https://github.com/anchore/grype#specifying-matches-to-ignore
+  # example to ignore a vulnerability
+  # This is the full set of supported rule fields:
+  # - vulnerability: CVE-2008-4318
+  #   fix-state: unknown
+  #   package:
+  #     name: libcurl
+  #     version: 1.5.1
+  #     type: npm
+  #     location: "/usr/local/lib/node_modules/**"
+
+# ignore postgres vulnerabilities
+- package:
+    type: apk
```

Comment:
are you sure we should ignore all APK vulns?
That seems like a large fraction of vulns...

Reply:
this one is mainly from base image of postgres and fix them will be a big task.

Reply:
that means "update the postgres base image", IMO